setup.cfg

```diff
@@ -17,38 +17,39 @@ classifiers =
   Intended Audience :: Science/Research
   License :: OSI Approved :: BSD License
   Natural Language :: English
-  Programming Language :: Python :: 3.5
-  Programming Language :: Python :: 3.6
   Programming Language :: Python :: 3.7
-  Operating System :: OS Independent
+  Programming Language :: Python :: 3.8
+  Programming Language :: Python :: 3.9
+  Operating System :: MacOS :: MacOS X
+  Operating System :: POSIX :: Linux
 license_file = LICENSE
 long_description = file: README.md
 long_description_content_type = text/markdown
 
-  [options]
-  python_requires = >= 3.5
-  setup_requires =
-    setuptools >= 38.6
-    pip >= 10
-  include_package_data = True
-  zip_safe = False
-  packages = find:
-  install_requires =
-    netCDF4
-    requests
-    beautifulsoup4
-    lxml
-    numpy
-    pandas
-    xarray
-    pysat
+[options]
+python_requires = >= 3.5
+setup_requires =
+  setuptools >= 38.6
+  pip >= 10
+include_package_data = True
+zip_safe = False
+packages = find:
+install_requires =
+  beautifulsoup4
```

Comment:
Do we still need to include packages that pysat requires?

Reply:
We shouldn't need these now that we're pulling pysat from pip